packages/calcite-design-tokens/src/semantic/space.json

```diff
@@ -0,0 +1,54 @@
+{
+  "semantic": {
+    "spacing": {
+      "px": {
+        "value": "{core.size.default.1}",
+        "type": "spacing"
+      },
+      "base": {
+        "value": "{core.size.default.2}",
+        "type": "spacing"
+      },
+      "xxs": {
+        "value": "{core.size.default.4}",
+        "type": "spacing"
+      },
+      "xs": {
+        "value": "{core.size.default.6}",
+        "type": "spacing"
+      },
+      "sm": {
+        "value": "{core.size.default.8}",
+        "type": "spacing"
+      },
+      "md": {
+        "value": "{core.size.default.12}",
+        "type": "spacing"
+      },
+      "lg": {
+        "value": "{core.size.default.14}",
+        "type": "spacing"
+      },
+      "xl": {
+        "value": "{core.size.default.16}",
+        "type": "spacing"
+      },
+      "xxl": {
+        "value": "{core.size.default.20}",
+        "type": "spacing"
+      },
+      "xxxl": {
+        "value": "{core.size.default.32}",
+        "type": "spacing"
+      },
+      "overlap": {
```

Comment:
For my own curiosity, what would be a typical use case for this token?

Reply:
I don't know. I copied it from Figma variables. Maybe @SkyeSeitz can give some clarity.

Reply:
Yup definitely not typical. Currently just used in Card for padding to the end of header. This is true in code and Figma.